consensus/istanbul/backend/backend.go

```diff
@@ -462,7 +462,8 @@ func (sb *Backend) GetValidators(blockNumber *big.Int, headerHash common.Hash) [
 	return validatorSet.List()
 }
 
-// Commit implements istanbul.Backend.Commit
+// Commit implements istanbul.Backend.Commit, unless this is committing the
+// last block of an epoch the aggregatedEpochValidatorSetSeal will be empty.
```

Comment:
I found this a bit hard top parse because I read it as a single sentence the first few times.
```suggestion
// Commit implements istanbul.Backend.Commit.
// Unless this is committing the last block of an epoch the aggregatedEpochValidatorSetSeal will be empty.
```